dspace-oai/src/main/java/org/dspace/utils/LangUtil.java

```diff
@@ -0,0 +1,112 @@
+/**
```

Comment:
I would rather move this away from org.dspace if possible

Reply:
There isn't anything else in this module, everything is in org.dspace. Should I spend more time on this?